nrt/data/DynaFlow/GeneratorPVTfo_rpcl_rpcl2/GeneratorPVTfo_rpcl_rpcl2.dyd

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:blackBoxModel id="SignalN" lib="Step" parFile="GeneratorPVTfo_rpcl_rpcl2.par" parId="SignalN"/>
+  <dyn:blackBoxModel id="DisconnectLine" lib="EventSetPointBoolean" parFile="GeneratorPVTfo_rpcl_rpcl2.par" parId="DisconnectLine"/>
+  <!-- System 3 with transformers and a secondary voltage control -->
```

Comment:
As we now only have two systems, could you rename them ? Same for the generators, lines, inf bus, etc.